src/messages/Image/Image.tsx

```diff
@@ -1,20 +1,22 @@
 import { FC, useMemo, useState } from "react";
-import { MessagePasstroughProps } from "../types";
 import { ImageMessageContext } from "./context";
 import Lightbox from "./lightbox/Lightbox";
 import ImageThumb from "./ImageThumb";
 import { useMessageContext } from "src/hooks";
+import { getChannelPayload } from "src/utils";
+import { IWebchatButton } from "@cognigy/socket-client/lib/interfaces/messageData";
 
-const Image: FC<MessagePasstroughProps> = () => {
-	const { message } = useMessageContext();
+const Image: FC = () => {
```

Comment:
why remove the interface?

Reply:
in the previous PR from Dmitrii we introduced the message context and we not passing any property to the component